<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ISeg_igst</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,4">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
         </font>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
         </font>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
         </font>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="spacing">
       <number>20</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_a" stretch="410,16">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QGraphicsView" name="view_a">
          <property name="minimumSize">
           <size>
            <width>410</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_a" stretch="18,280">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QToolButton" name="toolButton_a">
            <property name="minimumSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <pointsize>6</pointsize>
             </font>
            </property>
            <property name="text">
             <string>A</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>C:/Users/small/Downloads/相机.png</normaloff>C:/Users/small/Downloads/相机.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollBar" name="verticalScrollBar_a">
            <property name="minimumSize">
             <size>
              <width>18</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>128</number>
            </property>
            <property name="value">
             <number>64</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3d">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QGraphicsView" name="view_3d"/>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_c" stretch="410,16">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QGraphicsView" name="view_c">
          <property name="minimumSize">
           <size>
            <width>410</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_c" stretch="18,280">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QToolButton" name="toolButton_c">
            <property name="minimumSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <pointsize>6</pointsize>
             </font>
            </property>
            <property name="text">
             <string>C</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>C:/Users/small/Downloads/相机.png</normaloff>C:/Users/small/Downloads/相机.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollBar" name="verticalScrollBar_c">
            <property name="minimumSize">
             <size>
              <width>18</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>128</number>
            </property>
            <property name="value">
             <number>64</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_s" stretch="410,16">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QGraphicsView" name="view_s">
          <property name="minimumSize">
           <size>
            <width>410</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_s" stretch="18,280">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QToolButton" name="toolButton_s">
            <property name="minimumSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <pointsize>6</pointsize>
             </font>
            </property>
            <property name="text">
             <string>S</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>C:/Users/small/Downloads/相机.png</normaloff>C:/Users/small/Downloads/相机.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollBar" name="verticalScrollBar_s">
            <property name="minimumSize">
             <size>
              <width>18</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>128</number>
            </property>
            <property name="value">
             <number>64</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_main_image"/>
   </widget>
   <widget class="QMenu" name="menuseg">
    <property name="title">
     <string>Segmentation</string>
    </property>
    <addaction name="actionopen_seg"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuseg"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_main_image">
   <property name="text">
    <string>打开图像</string>
   </property>
  </action>
  <action name="actionopen_seg">
   <property name="text">
    <string>打开分割结果</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
